packages/flutter_tools/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   intl: '>=0.14.0 <0.16.0'
   json_rpc_2: ^2.0.0
   json_schema: 1.0.6
-  linter: 0.1.31
+  linter: 0.1.35
```

Comment:
I see [1.25.0-dev.8.0](https://github.com/dart-lang/sdk/blob/1.25.0-dev.8.0/DEPS) gets us to `0.1.33` but I think we'll need to wait for the next push to get to `0.1.35` since that bump to DEPS only landed in the Dart SDK this AM.

Sorry for the continued confusion!